tests/utils/test_authorization.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import unittest
+from app.database import db
+from app import create_app
+from app.utils.authorization import get_roles, roles
+from app.mod_api.models import MetaDataDB, User, Publisher, PublisherUser
+
+
+class AuthorizationTestCase(unittest.TestCase):
```

Comment:
This test only checks roles. We should actually check that some action performed by unauthorized users is denied.